recipes/scnlib/all/conanfile.py

```diff
@@ -18,11 +18,15 @@ class ScnlibConan(ConanFile):
         "header_only": [True, False],
         "shared": [True, False],
         "fPIC": [True, False],
+        "with_exceptions": [True, False],
+        "with_rtti": [True, False],
     }
     default_options = {
         "header_only": False,
         "shared": False,
         "fPIC": True,
+        "with_exceptions": True,
+        "with_rtti": True,
```

Comment:
I don't like these options, it's like adding a `lto` option just because upstream has decided to add an option just to add an option to CFLAGS.
From what I see, those 2 options have no logic except adding compile flags, so it's useless because if we go into this direction we should provide tones of options in each recipe to do some babysitting of compilers options.

Reply:
I understand your comment.
I agree with you.
I remove these options.